src/FS.Physical/PhysicalFilesWatcher.cs

```diff
@@ -21,12 +21,10 @@ namespace Microsoft.Extensions.FileProviders.Physical
     ///     Triggers events on <see cref="IChangeToken" /> when files are created, change, renamed, or deleted.
     ///     </para>
     /// </summary>
-    public class PhysicalFilesWatcher : IDisposable
+    internal sealed class PhysicalFilesWatcher : IFileWatcher
```

Comment:
Breaking changes for 3.0

Reply:
Why?

Reply:
Yea, we shouldn't do this.

Reply:
Ok. I assumed these were all internal implementation details that we should haven't made public and 3.0 would be a fair opportunity to change it. That said, we would have to change it's behavior \ API. Currently this type supports polling (it's a ctor parameter).